frame/support/procedural/src/pallet/expand/pallet_struct.rs

```diff
@@ -160,11 +160,15 @@ pub fn expand_pallet_struct(def: &mut Def) -> proc_macro2::TokenStream {
 		}
 	);
 
-	let storage_version = if let Some(v) = def.pallet_struct.storage_version.as_ref() {
-		quote::quote! { #v }
-	} else {
-		quote::quote! { #frame_support::traits::StorageVersion::default() }
-	};
+	let (storage_version, current_storage_version_ty) =
```

Comment:
nit: Does ‘ty’ stand for type? Probably best to do the whole word, ‘ty’ brings some other unrelated associations prior to investigating the code.

Reply:
That is a common abbreviation for type. You can also check syn or even other macro code.

Reply:
roger, haven’t noticed this before